plugins/completion/copilot-vim.nix

```diff
@@ -12,33 +12,70 @@ with helpers.vim-plugin;
     originalName = "copilot.vim";
     defaultPackage = pkgs.vimPlugins.copilot-vim;
     globalPrefix = "copilot_";
-    deprecateExtraConfig = true;
 
-    options = {
-      nodeCommand = mkDefaultOpt {
-        global = "node_command";
-        type = types.str;
-        default = "${pkgs.nodejs-18_x}/bin/node";
-        description = "Tell Copilot what `node` binary to use.";
-      };
+    # TODO introduced 2024-03-02: remove 2024-05-02
+    deprecateExtraConfig = true;
+    optionsRenamedToSettings = [
+      "nodeCommand"
+      "filetypes"
+      "proxy"
+    ];
 
-      filetypes = mkDefaultOpt {
-        type = with types; attrsOf bool;
-        description = ''
-          A dictionary mapping file types to their enabled status
+    settingsOptions = {
+      node_command = helpers.defaultNullOpts.mkStr "${pkgs.nodejs-18_x}/bin/node" ''
```

Comment:
I don't think this  is  the right type, because this won't set the "${pkgs.nodejs-18_x}/bin/node" as the nix default, I think we still  want that

Reply:
Oh yes you are right !

Reply:
Done